torch/testing/_internal/optests/generate_tests.py

```diff
@@ -193,7 +193,7 @@ def new_method(*args, **kwargs):
             construct_method(attr, prefix, tester)
 
 
-TEST_OPTIONS = ("xfail", "skip", "success")
+TEST_OPTIONS = ("xfail", "skip", "xsuccess")
```

Comment:
Just curious, is this just consistency with xfail? (and any reason not to do xskip?)

> Not BC breaking because no existing failures dict have "success" in
them.

also 🤣

Reply:
also try pronouncing it now lol

Reply:
Skip's a little different:

"xfail" = expected to fail
"xsuccess" = expected to succeed
"skip" = please skip this test